src/complex/indic.rs

```diff
@@ -593,7 +593,7 @@ fn collect_features(planner: &mut ShapePlanner) {
     planner.ot_map.enable_feature(feature::CONTEXTUAL_ALTERNATES, FeatureFlags::NONE, 1);
     planner.ot_map.enable_feature(feature::CONTEXTUAL_LIGATURES, FeatureFlags::NONE, 1);
 
-    planner.ot_map.add_gsub_pause(Some(ffi::rb_layout_clear_syllables));
+    planner.ot_map.add_gsub_pause(Some(crate::ot::rb_clear_syllables));
```

Comment:
Can we convert it to Rust function now? Or is it still used in C++?

Reply:
The GSUB pauses are only _called_ from rust, but still passed through C++ (in hb-ot-map). So, we could theoretically make them Rust functions, but it's a bit of a hack because we would need pass them around in C++ as void pointers or anything basically.

Reply:
Ok. I thought that we don't have to do this anymore.